.github/workflows/master.yaml

```diff
@@ -17,10 +17,6 @@ jobs:
         uses: actions/checkout@v2
       - name: Unshallow
         run: git fetch --prune --unshallow
-      - name: Set up Go
```

Comment:
https://github.com/actions/virtual-environments/blob/8491d71a0b76ae4a8572a27f683e396269bfa80c/images/linux/ubuntu1804.json#L27-L28 // https://github.com/actions/virtual-environments/blob/master/images/linux/Ubuntu1804-README.md

Looks like go 1.14 is included in the runner by default, so we get to skip a step here.

Reply:
Eh, added this back in. There's a bit of a lag ( 1.14.1 vs 1.14.2 )